docs/installation.md

```diff
@@ -101,19 +99,21 @@ You may want to configure a few additional components of Lychee, such as:
 - Database
 - Session
 
+They are documented in the `.env.example` file, which may have been copied into your `.env`. They are also listed on the [configuration](configuration.html) page.
+
 ##### Advanced Configuration (Dangerous)
 
-Some advanced options cannot be configured through the `.env` file, but you may take a look into the directory `config/` to find such options.
-Usually, you don't have to change them and can go with the defaults.
+Some advanced options cannot be configured through the `.env` file. If you really need to change them, look in the `config/` directory.
 
 **If you don't know what they're doing, do not change them.**
 
 ## Web Server Configuration
 
 ### General
 
-Configure your web server's root to be the `public` directory.
-The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API.
+Configure your web server to make the `public` directory accessible via a URL. This could be the root of your web server (short and convenient) or, if you prefer, a subfolder such as `photos`. The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API.
+
+> {note} Do not expose the top level directory with your web server! This would allow public access to internal files like the database and your `.env` which can leak your credentials!
```

Comment:
```suggestion
> {note} Do not expose the top level directory of Lychee with your web server! This would allow public access to internal files like the application code and the `.env` file which contains sensitive information like the database credentials!
```

Reply:
It *could* expose the DB (for sqlite), but yeah, I wasn't convinced about mentioning that either.